Runtime/DialogueRunner.cs

```diff
@@ -749,50 +768,65 @@ Dialogue.HandlerExecutionType HandleCommand(Command command)
                     }
                 }
 
-                // We didn't find it in the comand handlers. Try looking in the game objects.
+                // We didn't find it in the comand handlers. Try looking in
+                // the game objects.
                 (wasValidCommand, executionType) = DispatchCommandToGameObject(command);
 
                 if (wasValidCommand) {
                     // We found an object and method to invoke as a Yarn
-                    // command. It may or may not have been a coroutine; if it
-                    // was a coroutine, executionType will be
+                    // command. It may or may not have been a coroutine; if
+                    // it was a coroutine, executionType will be
                     // HandlerExecutionType.Pause, and we'll wait for it to
                     // complete before resuming execution.
                     return executionType;
                 }
 
-                // We didn't find a method in our C# code to invoke. Try invoking on the 
-                // publicly exposed UnityEvent.
+                // We didn't find a method in our C# code to invoke. Try
+                // invoking on the publicly exposed UnityEvent.
                 onCommand?.Invoke(command.Text);
                 return Dialogue.HandlerExecutionType.ContinueExecution;
             }
 
             /// Forward the line to the dialogue UI.
             Dialogue.HandlerExecutionType HandleLine(Line line)
             {
-                // Update lines to current state before sending them to the view classes
+                // Update lines to current state before sending them to the
+                // view classes
                 var substitutions = GetInlineExpressions(line);
                 var text = localizedText.ContainsKey(line.ID) ? localizedText[line.ID] : string.Empty;
                 var audio = localizedAudio.ContainsKey(line.ID) ? localizedAudio[line.ID] : null;
-                var localizedLine = new LocalizedLine() {
+
+                CurrentLine = new LocalizedLine()
+                {
                     TextID = line.ID,
                     TextLocalized = text,
                     VoiceOverLocalized = audio,
-                    Substitutions = substitutions
+                    Substitutions = substitutions,
+                    Status = LineStatus.Running
```

Comment:
It doesn't look like the `LineStatus` is actually being used anywhere later on (like view classes updating the `LineStatus`) or if it's possible to retrieve the status of a line. It rather seems to me that the `LineStatus` enum is used as a way of sending specific notifications to the views via `OnLineStatusChanged()`. So we could either remove the `Status`variable from the `LocalizedLine` class and only use it for sending notifications to the view classes as we currently do or we actually change the LineStatus in the view classes and make that status retrievable should we think it could be useful in the future of for the users of YarnSpinner.

Reply:
I think the change to `OnLineStatusChanged` makes leaving the Status field public a reasonable thing.